TrafficCapture/trafficReplayer/src/test/java/org/opensearch/migrations/replay/V0_1TrafficCaptureSource.java

```diff
@@ -7,17 +7,21 @@
 import org.opensearch.migrations.replay.traffic.source.TrafficStreamWithEmbeddedKey;
 import org.opensearch.migrations.trafficcapture.protos.TrafficStream;
 
+import java.io.EOFException;
 import java.io.FileInputStream;
 import java.io.IOException;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Optional;
 import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.atomic.AtomicInteger;
 import java.util.stream.Collectors;
 import java.util.zip.GZIPInputStream;
 
 public class V0_1TrafficCaptureSource implements ISimpleTrafficCaptureSource {
 
+    public static final int NUM_TRAFFIC_STREAMS_TO_READ = 1 * 1000;
```

Comment:
Curious why "1*1000"?

Reply:
I've been playing around with changing that value from anything between 1K and 1M.  Having three zeros in a row is my limit of visual perception.  Yeah - this could just be 1K, but I'll often change it to 10K, 100K, 1000K, etc (I didn't think about the redundancy when I knocked it down to 1K).